docs/agents/labjack.rst

```diff
@@ -41,32 +41,36 @@ You should assign your LabJack a static IP, you'll need to know that here.
 The 'active-channels' argument specifies the channels that will be read out.
 It can be a list, 'T7-all', or 'T4-all'. The latter two read out all 
 14 or 12 analog channels on the T7 and T4, respectively. 'sampling_frequency'
-is in Hz, and has been tested sucessfully up to about 700 Hz. The 'function-file' 
-argument specifies the labjack configuration file, which is located in your 
-OCS configuration directory. This allows analog voltage inputs on the labjack 
-to be converted to different units. Here is an example labjack configuration 
-file::
-
-    Channel_1:
+is in Hz, and has been tested sucessfully from 0.1 to 5000 Hz. To avoid 
+high sample rates potentially clogging up live monitoring, the main feed 
+doesn't get published to influxdb. Instead influx gets a seperate feed 
+downsampled to a maximum of 1Hz. Both the main and downsampled feeds are 
+published to g3 files. 
+
+The 'function-file' argument specifies the labjack configuration file, which 
+is located in your OCS configuration directory. This allows analog voltage 
+inputs on the labjack to be converted to different units. Here is an example 
+labjack configuration file::
+
+    AIN0:
         user_defined: 'False'
         type: "MKS390"
 
-    Channel_2: 
+    AIN1: 
         user_defined: 'False'
         type: 'warm_therm'
 
-    Channel_3:
+    AIN2:
         user_defined: 'True'
         units: 'Ohms'
         function: '(2.5-v)*10000/v'
         
-In this example, Channels 1 and 2 (AIN0 and AIN1 on the labjack) are hooked
-up to the MKS390 pressure `gauge`_ and a `thermistor`_ from the SO-specified
-warm thermometry setup, respectively. Since these are defined functions in the
-LabJackFunctions class, specifying the name of their method is all that is
-needed. Channel 3 shows how to define a custom function. In this case, the user
-specifies the units and the function itself, which takes the input voltage 'v'
-as the only argument.
+In this example, channels AIN0 and AIN1 are hooked up to the MKS390 pressure 
+`gauge`_ and a `thermistor`_ from the SO-specified warm thermometry setup, 
+respectively. Since these are defined functions in theLabJackFunctions class, 
```

Comment:
"theLabJackFunctions" -> "the LabJackFunctions" (missing space)

Reply:
fixed